### PR TITLE
added section for Generate PR Description

### PR DESCRIPTION
**Pull request by vim-zz**
Screenshot 2024-08-08 at 18 28 55

### CODE REVIEW

**Comment by MishaKav** [comment]
👍

File: docs/downloads/automation-library/auto_pr_description.cm
Comment (BenLloydPearson) [inline docs/downloads/automation-library/auto_pr_description.cm:1]: Rename this to match the docs directory name and update the reference in the README. This file should be named `generate_pr_description.cm`

File: docs/downloads/automation-library/auto_pr_description.cm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+triggers:
```
Comment (BenLloydPearson) [inline docs/downloads/automation-library/auto_pr_description.cm:1]: Should this also have a trigger to only run when the PR is created or has new commits pushed?

**Review by BenLloydPearson — APPROVED** [review]
I left some comments, but otherwise this lgtm.

**Pull request merged**